src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -138,26 +145,47 @@
          to one of its options).  Simply appending the traits would break code coverage. -->
     <PropertyGroup>
       <TestCommandLine>$(TestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</TestCommandLine>
+      <BareTestCommandLine>$(BareTestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</BareTestCommandLine>
```

Comment:
When the .sh file version works, I expect to just use Bare (and probably just rename it to TestCommandLine).   I only kept both to try to leave UNIX execution unaffected.